### PR TITLE
Issue 6096 - Improve connection timeout error logging

### PR DESCRIPTION
**Pull request by jchapma**
Bug description: When a paged result search is run with a time limit, if the time limit is exceed the server closes the connection with closed IO timeout (nsslapd-ioblocktimeout) - T2. This error message is incorrect as the reason the connection has been closed was because the specified time limit on a paged result search has been exceeded.

Fix description: Correct error message

Relates: https://github.com/389ds/389-ds-base/issues/6096

Reviewed by:

### CODE REVIEW

File: ldap/admin/src/logconv.pl [hunk 1]
```diff
@@ -355,6 +355,7 @@
 $connmsg{"B4"} = "Server failed to flush data (response) back to Client";
 $connmsg{"T1"} = "Idle Timeout Exceeded";
 $connmsg{"T2"} = "IO Block Timeout Exceeded or NTSSL Timeout";
+$connmsg{"T3"} = "Paged Search Limit Exceeded";
```
Comment (jchapma) [inline ldap/admin/src/logconv.pl]: I thought it was best to another another T version, for T2 there are already 2 timeout reasons.

Comment (tbordaz) [reply ldap/admin/src/logconv.pl]: Right good idea. I would replace 'Search Limit' with 'Search Time Limit'

File: ldap/servers/slapd/disconnect_error_strings.h [hunk 2]
```diff
@@ -27,6 +27,7 @@ ER2(SLAPD_DISCONNECT_BER_FLUSH, "Server failed to flush response back to Client
 ER2(SLAPD_DISCONNECT_IDLE_TIMEOUT, "Idle Timeout (nsslapd-idletimeout) - T1")
 ER2(SLAPD_DISCONNECT_REVENTS, "Poll revents - R1")
 ER2(SLAPD_DISCONNECT_IO_TIMEOUT, "IO Block Timeout (nsslapd-ioblocktimeout) - T2")
+ER2(SLAPD_DISCONNECT_PAGED_SEARCH_LIMIT, "Paged Search Limit Exceeded - T3")
```
Comment (tbordaz) [inline ldap/servers/slapd/disconnect_error_strings.h]: 'Search Time Limit' would be more precise as there is also size limit that could be confusing

**Review by tbordaz — APPROVED** [review]
LGTM

File: ldap/servers/slapd/disconnect_error_strings.h (hunk 2 above)
Comment (jchapma) [reply ldap/servers/slapd/disconnect_error_strings.h]: Thats a good point, I will update

**Pull request merged**